src/Core/Domain/Entities/Automobile.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AspNetCoreSpa.Domain.Entities
+{
+    public class Automobile
+    {
+        public string AutomobileId { get; set; }
```

Comment:
String is not a good type for a primary key. Use autogenerated Guid or Int instead